ml-dhs-aws/src/main/java/com/marklogic/client/dhs/JobRunner.java

```diff
@@ -41,7 +42,8 @@ public String getJobDirectory() {
       return jobDirectory;
    }
 
-   public void run(DatabaseClient client, InputStream csvRecords, InputStream jobControl) throws IOException {
+   @SuppressWarnings("deprecation")
```

Comment:
Which API is deprecated? Is there a non-deprecated alternative?

Reply:
ObjectNode.put(String fieldName, JsonNode value) was outdated. Changed it to ObjectNode.set(String fieldName, JsonNode value).